cloud/aws/services/ec2/tags.go

```diff
@@ -28,9 +28,17 @@ const (
 	// The tag value is an ownership value
 	TagNameKubernetesClusterPrefix = "kubernetes.io/cluster/"
 
+	// TagNameAWSProviderManaged is the tag name we use to differentiate
+	// cluster-api-provider-aws owned components from other tooling that
+	// uses TagNameKubernetesClusterPrefix
+	TagNameAWSProviderManaged = "sigs.k8s.io/cluster-api-provider-aws/managed"
+
 	// TagNameAWSClusterAPIRole is the tag name we use to mark roles for resources
 	// dedicated to this cluster api provider implementation.
 	TagNameAWSClusterAPIRole = "sigs.k8s.io/cluster-api-provider-aws/role"
+
+	TagValueCommonRole    = "common"
+	TagValueAPIServerRole = "apiserver"
```

Comment:
Should we have a godoc string around these?

Reply:
Done